supply-chain-security/supply-chain-security-paper/sscsp.md

```diff
@@ -883,7 +877,7 @@ TUF has also been used to bootstrap trust in delivering software supply chain me
 
 **What is sigstore?**
 
-[https://sigstore.dev/what_is_sigstore/](https://sigstore.dev/what_is_sigstore/)
+[https://docs.sigstore.dev/](https://docs.sigstore.dev/)
```

Comment:
The old https://sigstore.dev/what_is_sigstore/ page no longer exists.  I asked the sigstore group if this page simply moved somewhere, but if this tweak isn't an issue it can go in as-is.